.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,40 @@
+name: "Run Tests"
+
+on:
+  pull_request:
```

Comment:
Yeah, it can be either a string, an array, or a key. I was originally playing with some options, but removed them.

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on